data_architect_misc/data_transformer/constants/comp_harm_constants.py

```diff
@@ -159,6 +159,14 @@
     "(?i)Cleaning.*agent.*": "Home Care",
     "(?i).*Detergent.*": "Home Care",
 
+    "(?i).*SOAP.*": "Personal Care",
```

Comment:
Don Maicol, can you please sort these mappings by alphabetical order (of the keys in the dictionary)? That way, it's easy for someone to scan them if we need to add new items to this mapping. Thank you.